### PR TITLE
Featuremapper now builds in Python 3.8.

### PR DESCRIPTION
**Pull request by fcr**
Builds in Python 3.8 include all the packages.
Increased version to 0.2.1

### CODE REVIEW

File: featuremapper/__init__.py [hunk 1]
```diff
@@ -470,12 +470,12 @@ def _update(self, p, current_values):
...
 
     @bothmethod
-    def set_cmd_overrides(self_or_cls, **kwargs):
+    def set_cmd_overrides(self, **kwargs):
         """
         Allows setting of cmd_overrides at the class and instance level.
         cmd_overrides are applied to the pattern_response_fn.
         """
-        self_or_cls.cmd_overrides = dict(self_or_cls.cmd_overrides, **kwargs)
+        self.cmd_overrides = dict(self.cmd_overrides, **kwargs)
```
Comment (jbednar) [inline featuremapper/__init__.py:478]: This change seems like a regression to me.  The parameter name doesn't matter to Python, but if it's a `bothmethod`, the first argument can either be an instance (normally called `self`) or a class (here called `cls`).  So this change doesn't hurt, but it does make it more confusing!

**Review by jbednar — COMMENTED** [review]
Thanks!

File: featuremapper/__init__.py (hunk 1 above)
Comment (fcr) [reply featuremapper/__init__.py:478]: I understand the original motivation, but some Python editors flag the original argument as an error. Besides self can semantically mean either the instance or the class depending on what is passed.

Comment (jbednar) [reply featuremapper/__init__.py:478]: I guess I don't agree that `self` can mean "class" in Python; `self` being anything other than an instance object is highly weird! Plus it sounds like the problem here is with an editor that doesn't understand `bothmethod`, not with the code itself. Still, at least that pragmatic consideration does explain why the change was made, thanks!

Comment (fcr) [reply featuremapper/__init__.py:478]: If you think of a class as an instance of its metaclass than it is not so weird.

**Pull request merged**